### ISSUE TITLE
Compiling downloaded zip

### ISSUE THREAD

**Issue by kowalsk** [pupi1985/IPMonitor#2]
Hi, Sorry not sure how else to ask question here - I am having difficulty running your project in Eclipse, probably because it is on a PC and I think your version is from a Linux/Mac. Is there any chance you could explain how to properly import this project into Eclipse or how else to compile and run it on a PC? I am a bit new to Java world but your project seems very interesting.
This is the error message I get:
Error: Could not find or load main class main.Main

**Comment by pupi1985** [comment]
Hi,

I'm in Linux but that doesn't change anything. In order to setup the project in Eclipse you could:
1. Download the source code or clone the Git repository
2. Open Eclipse and import the project you've just downloaded
3. In the project properties, look for section Builders and make sure the Ant builder is checked (it should by default)
4. In the Project menu click Build All
5. Press F5 in the project tree to refresh and see the newly generated directory (or navigate to the project directory using your OS)
6. In the new `dist` directory you should see all the needed files to run IP Monitor

The `build.xml` file contains all the steps needed in the build process, if you are interested in them. This process is OS independent.

**Comment by kowalsk** [comment]
OK great, this has finally worked. It looks like my Java was not fully installed - it worked with other projects and small app I created to test but not with yours. I reinstalled Java, changed PATH once again, this time moved Java path at the end of PATH variable which seems to have made a difference. Anyway, works now, thank for your time! It is very frustrating for beginners to find things don't work and you were kind enough to confirm it is indeed OK, thanks for that.
